src/openrct2/ride/TrackData.cpp

```diff
@@ -5255,6 +5269,10 @@ const money32 FlatRideTrackPricing[] = {
     151552,
     151552,
     151552,
+    360448, // TrackElemType::MultiDimUp90ToInvertedFlatQuarterLoop
+    360448, // TrackElemType::MultiDimFlatToDown90QuarterLoop
+    360448, //
+    77824,  // TrackElemType::RotationControlToggle
```

Comment:
Set the size of this array to `FlatRideTrackPricing`, so we'll know if it contains too few entries. Same for the other ones.